Petar_Nedyalkov/week2/task2.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <fstream>
+
+int countCertainDigits(std::ifstream& file)
+{
+	int count = 0;
+	while (!file.eof())
+	{
+		if (file.get() != 'a' || file.get() != 'd') count++;
```

Comment:
`file.get` reads a new symbol every time. This `if` will not do what you expect out of it :P